unity/CITools/BuildDriver/Program.cs

```diff
@@ -163,8 +163,10 @@ void Run(InvocationContext context)
             {
                 if (bTargets.HasFlag(BuildTargets.NullGC))
                     NullGC.Build(gConfig);
-                if (bTargets.HasFlag(BuildTargets.CoreCLR))
-                    CoreCLR.Build(gConfig);
+
+                var subsets = bTargets.ToSubSetString();
+                if (!string.IsNullOrEmpty(subsets))
```

Comment:
I think we should pass the enum bitmask into CoreCLR and let the parsing happen there so it's in one place.